app/models/subcategory.rb

```diff
@@ -1,4 +1,5 @@
-class QuestionType < ApplicationRecord
```

Comment:
This causes an issue with the migrations. If someone checks out the code and needs to run the first migration in this PR, it breaks because the QuestionType class no longer exists. I recommend copying the model into the migration.